UX-Guide-Metadata/draft/principles/index.html

```diff
@@ -1101,17 +1101,17 @@ <h3 data-localization-id="legal-considerations-title">Legal considerations</h3>
 					is to provide as much accessibility information as is possible.</p>
 
 				<section id="legal-examples">
-					<h4>Examples</h4>
-					<p>TBD</p>
-					<!--
+					<h4>Example</h4>
+
+					
+<p>In many cases no legal exception or exemption information will be presented. Where it it is provided, the example is  provided as   a possible descriptive   explanation.</p>
+ 
 					<aside class="example" title="Descriptive explanation">
```

Comment:
It might be a bit strange to have a singular "Example" title for the section followed by a description of the example followed by the example box which will generate another example heading. Usually when a section is an example you don't use a box. If you want to keep this formatting, I'd put the description for the example inside the box, as well.